rust/matches/src/utils.rs

```diff
@@ -421,3 +423,16 @@ pub fn is_valid_validation<'info>(
 
     return Ok(true);
 }
+
+// returns true if the namespace program called the item program
+pub fn is_namespace_program_caller(ixns: &AccountInfo) -> bool {
```

Comment:
This should probably also be in a global raindrops utils package. Or at least in the namespace package. Since the namespace package is who knows the definition of the namespace program

Reply:
It cant be in the namespace package because it would create a dep cycle between the 2 programs, needs to be in a "commons" crate

Reply:
Commons it is